src/main/java/mekanism/common/item/interfaces/IItemHUDProvider.java

```diff
@@ -9,4 +9,8 @@
 public interface IItemHUDProvider {
 
     void addHUDStrings(List<Component> list, Player player, ItemStack stack, EquipmentSlot slotType);
+
+    default void addCurioHUDStrings(List<Component> list, Player player, ItemStack stack) {
+
```

Comment:
Blank line for the body can be removed (keep the closing brace on its own line though)